### PR TITLE
Update proteinortho to 6.0.7

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`proteinortho`](https://bioconda.github.io/recipes/proteinortho/README.html): **6.0.6** &rarr; **6.0.7**

[install with bioconda](http://bioconda.github.io/recipes/proteinortho/README.html) [Conda](https://anaconda.org/bioconda/proteinortho/files)

Info | Link
-----|-----
Recipe | [`recipes/proteinortho`](https://github.com//bioconda/bioconda-recipes/tree/bump/proteinortho/recipes/proteinortho) (click to view/edit other files)
Releases |[https://gitlab.com/paulklemm_PHD/proteinortho/tags](https://gitlab.com/paulklemm_PHD/proteinortho/tags)

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by pmjklemm** [comment]
I was not expecting this to fail, i will test now some variations of the configuration files

**Comment by pmjklemm** [comment]
so with cmake proteinortho can recompile a lapack version. (apperently the dynamic lapack fails with this version idk)

**Comment by pmjklemm** [comment]
now i try to get external lapack working (via meta.yaml)

**Comment by pmjklemm** [comment]
-lapack works for linux but fails for mac, so now i try -lapack and -openblas

**Comment by pmjklemm** [comment]
that is weird @dpryan79 :
06:46:39 BIOCONDA INFO (ERR)   Reason: Incompatible library version: proteinortho_clustering requires version 3.0.0 or later, but liblapack.3.dylib provides version 0.0.0

(only mac fails with lapack ....)

**Comment by pmjklemm** [comment]
So the problem is:

the lapack library included with meta.yaml compiles perfectly fine under linux and mac ! But mac claims that lapack v3 is version 0.0.0 ? 
It works with my own lapack delivered with this program (BUT this should only used as fallback, since I do not update this version of lapack)

**Comment by dpryan79** [comment]
I don't understand that error message either :( Maybe this can be raised on the conda-forge lapack feedstock.

**Comment by pmjklemm** [comment]
I did wrote a mail to conda-forge@googlegroups.com, I will write an update when I get a response.

File: recipes/proteinortho/meta.yaml [hunk 1]
```diff
@@ -20,13 +20,13 @@ requirements:
     - cmake
   host:
     - openmp
-    - lapack
```
Comment (isuruf) [inline recipes/proteinortho/meta.yaml]: This package is deprecated. Use `liblapack`.

Comment (pmjklemm) [reply recipes/proteinortho/meta.yaml]: lapack does not work:

mac runtime-error:
`06:46:39 BIOCONDA INFO (ERR) Reason: Incompatible library version: proteinortho_clustering requires version 3.0.0 or later, but liblapack.3.dylib provides version 0.0.0`

But liblapacke works ... idk

**Pull request merged**